src/screens/TransactionDetailsScreen.tsx

```diff
@@ -168,8 +179,41 @@ class TransactionDetailsScreen extends Component<Props, State> {
     });
   };
 
+  renderLabel = (txType: TxType) => {
```

Comment:
this code is duplicated from the transaction item, make the component something like <TransactionLabel txType={txType} /> and use it here and in the transaction item